Resources/Prototypes/Entities/Objects/Weapons/Guns/Pistols/pistols.yml

```diff
@@ -64,12 +64,11 @@
   name: viper
   parent: BaseWeaponPistol
   id: WeaponPistolViper
-  description: A small, easily concealable, but somewhat underpowered gun. Retrofitted with a fully automatic receiver. Uses .35 auto ammo.
+  description: A small, easily concealable, but somewhat underpowered gun, produced by a bulk arms manufacturer now defunct for over a century. Uses .35 auto ammo.
   components:
   - type: Sprite
     sprite: Objects/Weapons/Guns/Pistols/viper.rsi
     availableModes:
-    - FullAuto
```

Comment:
`# DeltaV - Prevent Viper from being full auto`
Also comment this line don't delete it